.github/workflows/lint.yml

```diff
@@ -26,3 +26,6 @@ jobs:
       - run: npm ci
       - name: Lint
         run: npm run lint
+
+      - name: Verify READMEs
+        run: npm run readme:verify
```

Comment:
nit: Perhaps call it `lint:readme` to fit in with lint:markdown, lint:examples?

Reply:
I like it. updated the PR